components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -47,10 +46,9 @@
               <chef-td>
                 <a [routerLink]="['/infrastructure/chef-servers', server.id]">{{ server.name }}</a>
               </chef-td>
-              <chef-td>{{ server.fqdn }}</chef-td>
-              <chef-td>{{ server.ip_address }}</chef-td>
-              <chef-td>{{ server.description }}</chef-td>
-              <chef-td>{{ server.orgs_count }}</chef-td>
+              <chef-td  >{{ server.fqdn }}</chef-td>
+              <chef-td  >{{ server.ip_address }}</chef-td>
+              <chef-td  >{{ server.orgs_count }}</chef-td>
```

Comment:
@vinay033 nitpick of removing added space in <chef-td    >

Reply:
fixed